tsconfig.base.json

```diff
@@ -17,7 +17,8 @@
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
     "paths": {
-      "@lit-protocol/*": ["packages/*/src"]
+      "@lit-protocol/*": ["packages/*/src"],
+      "@lit-protocol/pkp-sui": ["packages/pkp-sui/src/index.ts"]
```

Comment:
don't need `"@lit-protocol/pkp-sui": ["packages/pkp-sui/src/index.ts"]` as `"@lit-protocol/*": ["packages/*/src"],` is using wildcard